README.md

```diff
@@ -38,115 +34,67 @@
-4. Push to the Branch (`git push origin feature/AmazingFeature`)
-5. Open a Pull Request
-
-
-
-<!-- LICENSE -->
-## License
-
-Distributed under the MIT License. See `LICENSE` for more information.
+* Here's something we'd like to see added
+* Here's something we'd like to see added
+* Here's something we'd like to see added
 
+See the [open issues](https://github.com/ericberglund117/Refactor-tractor-WC-EB-KM-CD/issues) for a list of proposed features (and known issues).
 
 
 <!-- CONTACT -->
 ## Contact
 
-Your Name - [@twitter_handle](https://twitter.com/twitter_handle) - email
-
-Project Link: [https://github.com/github_username/repo_name](https://github.com/github_username/repo_name)
+Eric Berglund - [Eric's Github Link](https://github.com/ericberglund117)
 
+Kristi Miller - [Kristi's Github Link](https://github.com/Kristiannmiller)
 
+Chadrick Dickerson - [Chadrick's Github Link](https://github.com/chadrick-d-dev)
 
-<!-- ACKNOWLEDGEMENTS -->
-## Acknowledgements
+Project Link: [https://github.com/ericberglund117/Refactor-tractor-WC-EB-KM-CD](https://github.com/ericberglund117/Refactor-tractor-WC-EB-KM-CD)
 
-* []()
-* []()
-* []()
 
 
+<!-- ****************** INFO NEEDED **************** -->
+## Acknowledgements
 
+Robert Gu - This developer team would like to thank Turing Module 2 anchor instructor Robert Gu for his assistance throughout this project and for his daily lessons on the magic of Javascript.
 
+Travis Rollins - We would also like to thank Turing Module 2 instructor Travis Rollins for his guidance and assistance on this project and the module in general
 
-<!-- MARKDOWN LINKS & IMAGES -->
-<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
-[contributors-shield]: https://img.shields.io/github/contributors/github_username/repo.svg?style=flat-square
-[contributors-url]: https://github.com/github_username/repo/graphs/contributors
-[forks-shield]: https://img.shields.io/github/forks/github_username/repo.svg?style=flat-square
-[forks-url]: https://github.com/github_username/repo/network/members
-[stars-shield]: https://img.shields.io/github/stars/github_username/repo.svg?style=flat-square
-[stars-url]: https://github.com/github_username/repo/stargazers
-[issues-shield]: https://img.shields.io/github/issues/github_username/repo.svg?style=flat-square
-[issues-url]: https://github.com/github_username/repo/issues
-[license-shield]: https://img.shields.io/github/license/github_username/repo.svg?style=flat-square
-[license-url]: https://github.com/github_username/repo/blob/master/LICENSE.txt
-[linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=flat-square&logo=linkedin&colorB=555
-[linkedin-url]: https://linkedin.com/in/github_username
-[product-screenshot]: images/screenshot.png
+Our Turing Mentors - Including Turing-wide mentor John Adams, and Turing graduates Jon Sweet, Pol Sieira, and Aimee Fritz for their constant support throughout this challenging project.
```

Comment:
I really appreciate the README update! It looks great so far. This will make it much easier towards the end of the project

Reply:
Yeah this is very thoughtful, I like this addition as well.